openreview/venue/webfield/programChairsWebfield.js

```diff
@@ -1,42 +1,61 @@
 // Webfield component
+const automaticAssignment = domain.content.automatic_reviewer_assignment?.value
+const assignmentUrls = {}
+
+const manualReviewerAssignmentUrl = `/edges/browse?traverse=${domain.content.reviewers_assignment_id?.value}&edit=${domain.content.reviewers_assignment_id?.value};${domain.content.reviewers_custom_max_papers_id?.value},tail:ignore&browse=${domain.content.reviewers_affinity_score_id?.value};${domain.content.reviewers_conflict_id?.value}&version=2`
```

Comment:
if .value could be undefined, the link will be invalid

Reply:
they shoulnd't be undefined, they will be present in the domain object. I'm adding the `?` per Zach's recommendation.